src/content_mapblock.cpp

```diff
@@ -30,22 +30,36 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "log.h"
 #include "noise.h"
 
+#define SMOOTH_LIGHTING_OVERSIZE 0.0f
```

Comment:
This is intended for oversized nodes. It’s the distance the light value being extrapolated linearly, being constant at greater distance from the (logical) node boundary.
Not sure how to make a good comment from that, though.